src/billing/components/Free/OrgLimits.tsx

```diff
@@ -1,78 +1,110 @@
+// Libraries
 import React, {FC} from 'react'
 import {useSelector} from 'react-redux'
-import {Grid, Columns} from '@influxdata/clockface'
+import {Columns, Grid} from '@influxdata/clockface'
+
+// Components
 import OrgLimitStat from 'src/billing/components/Free/OrgLimitStat'
 
+// Selectors
+import {isOrgIOx} from 'src/organizations/selectors'
+
 // Types
 import {AppState} from 'src/types'
 import {Limit} from 'src/cloud/actions/limits'
 import {LimitsState} from 'src/cloud/reducers/limits'
 
-const limits = (orgLimits: LimitsState | {}) =>
+interface LimitCardProps {
+  limitName: string
+  limitValue: Limit
+}
+
+const removeUnrenderedProps = (orgLimits: LimitsState | {}) =>
   Object.entries(orgLimits).filter(
     ([limitName]) =>
       limitName !== 'orgID' &&
       limitName !== 'status' &&
       limitName !== 'endpoints'
   )
 
-const OrgLimits: FC = () => {
+const LimitCard: FC<LimitCardProps> = ({limitName, limitValue}) => {
+  return (
+    <Grid.Column
+      key={limitName}
+      widthXS={Columns.Twelve}
+      widthSM={Columns.Six}
+      widthMD={Columns.Four}
+      widthLG={Columns.Three}
+    >
+      <OrgLimitStat name={limitName} value={limitValue.maxAllowed} />
+    </Grid.Column>
+  )
+}
+
+export const OrgLimits: FC = () => {
   const orgLimits = useSelector((state: AppState) => state.cloud?.limits ?? {})
+  const orgUsesIOx = useSelector(isOrgIOx)
 
   return (
     <Grid>
-      {limits(orgLimits).flatMap(([name, value]: [string, Limit]) => {
-        if (name === 'buckets') {
-          return Object.entries(value)
-            .filter(([n]) => n !== 'limitStatus')
-            .map(([n, v]: [string, number]) => {
+      {removeUnrenderedProps(orgLimits).map(limit => {
+        const limitName = limit[0]
+        const limitValue = limit[1]
```

Comment:
alternatively, the new meta is:

```ts
const [limitName, limitValue] = limit
```

but who cares. It's a small stylistic thing. It's just that the kids love their destructuring, fr fr no cap

Reply:
Nah, you're right I should destructure this, it's more readable.